site-src/concepts/conformance.md

```diff
@@ -126,6 +126,10 @@ feature and any relevant API features, e.g:
 go test ./conformance/... -args -supported-features=Mesh,Gateway,HTTPRoute
 ```
 
+If the mesh you are testing requires labels on namespaces that host mesh workloads,
```

Comment:
Mind making this less mesh-specific? (I agree that this is probably most relevant for mesh implementations, but this could be useful more broadly, for example to simplify manual cleanup of resources or if a different kind of implementation also benefited from namespace labels).

Reply:
done.